pkg/daemon/update.go

```diff
@@ -970,6 +970,23 @@ func (dn *Daemon) updateOnClusterBuild(oldConfig, newConfig *mcfgv1.MachineConfi
 		return err
 	}
 
+	// Determine the pool (worker or master) based on the node's labels
```

Comment:
Question: Couldn't this be custom pool (ie non-{worker/master}), too?

Reply:
MCN technically doesn't yet support custom pools: https://issues.redhat.com/browse/MCO-1299